ipi/inputs/barostats.py

```diff
@@ -95,6 +95,14 @@ class InputBaro(Input):
                 "help": "A list of the cell entries that should be held fixed (xx, yy, zz, xy, xz, yz). 'offdiagonal' is an alias for xy, xz, yz.",
             },
         ),
+        "vol_constraint": (
+            InputValue,
+            {
+                "default": False,
+                "dtype": bool,
+                "help": "If True, a different ensemble is sampled where the cell volume remains fixed while allowing for full cell fluctuations.",
+            },
+        ),
```

Comment:
Here we need more documentation, otherwise the user has to go into the code to see what it means. Also, there should be a catch in the code to stop the code if someone tries to set it to true with a barostat that is not MTTK. 

Could you write this also in this description? Say what is the reference publication, what ensemble it is - NVsigmaT - , and that it only does something if coupled to MTTK. Also warn the code will stop if another barostat is chosen.